### ISSUE TITLE
Ray of Enfeeblement gets an incorrect stringref on BG:EE

### ISSUE THREAD

**Issue by luc-j-bourhis** [Gibberlings3/SpellRevisions#129]
When I cast it, the console prints: "Nonsense! One day you'll bring her home a gem the size of your fist and she'll change her mind.", which is string ref 7924. I think the source of the problem is that in BG2:EE (and I guess in BG2 too but I can't check), that string ref is "Enfeebled". But in BG:EE, that string has ref 25103 instead from a quick look with NearInfinity. Really strange they have different ref's!

**Comment by lambda-dom** [comment]
Yes, your analysis is entirely correct; the problem is due to mismatch tlk string references in the different games. It is a veritable PITA to fix these. Anyway, patch already in, so closing this.

**Comment by luc-j-bourhis** [comment]
Thanks. I actually had a very naive fix on my fork (link below) that I did not dare to create a pull request for! It worked… but reviewing it, I realise that I had discovered another string mismatch (and fixed it in the same naive way): 4742 is ~Spell Ineffective~ in BG2 but "Well, I never! There was no reason to get so snotty. I apologize for taking so much time out of your important lives!" in BG1. 

https://github.com/luc-j-bourhis/SpellRevisions/commit/47f3293a852a35e932e11edb5d56990ca4aafd3a
